edb/server/protocol/auth_ext/util.py

```diff
@@ -76,3 +82,15 @@ def get_config(
             description="Missing configuration value",
         )
     return value
+
+
+def get_config_unchecked(
```

Comment:
I didn't end up using this, but seems like a nice function to have anyway, so I left it. 🤷